arrow/util.py

```diff
@@ -37,4 +37,22 @@ def isstr(s):
         return isinstance(s, str)
 
 
-__all__ = ['total_seconds', 'isstr']
+def astimezone(date, tz):
+    ''' Returns a ``datetime`` object, adjusted to the specified tzinfo.
+
+        :param date: a ``datetime`` object.
+        :param tz: a ``tzinfo`` object.
+
+    '''
+
+    date = date.astimezone(tz)
+
+    if hasattr(tz, 'normalize'):
+        # A pytz function
+        # See http://pytz.sourceforge.net/#introduction
+        date = tz.normalize(date)
```

Comment:
This is not necessary - `astimezone(tz)` should always produce an unambiguous date, even with `pytz`. You only need to use `normalize` if you have done an arithmetical operation, in case this has resulted in the date crossing a boundary.